test/FunctionalTests/Microsoft.OData.Core.Tests/ServiceProviderExtensionsTests.cs

```diff
@@ -6,84 +6,82 @@
 
 using System;
 using System.Linq;
-using Microsoft.Test.OData.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection;
 using Xunit;
 
 namespace Microsoft.OData.Tests
 {
     public class ServiceProviderExtensionsTests

```

Comment:
None of these methods test relevant OData functionality. The file should probably be deleted.